raiden/ui/app.py

```diff
@@ -363,6 +341,11 @@ def run_app(
         secret_registry_contract_address is not None and
         endpoint_registry_contract_address is not None
     )
+    if environment_type == Environment.DEVELOPMENT:
+        contract_addresses_given = (
```

Comment:
Shouldn't this check also depend on the routing mode? If basic routing mode is used there is no need to know the service registry address.

Reply:
Well at the moment the two are tied together. But yes I guess this should depend on the routing mode in the long run.